WLC/web_endpoint.py

```diff
@@ -78,11 +78,29 @@ def api_resubmit_code():
         azure.save_code_to_azure('code', 'pictures', key, code)
 
         return json.dumps({'result': str(result), 'errors': errors, 'ar': ar, 'key': key,
-                           'test_results': test_results})
+                           'testResults': test_results})
```

Comment:
Why cammel case when we use underscores everywhere?

Reply:
Because from what I have seen json objects usually have camel cased keys.

Reply:
Hmm I guess, we should have set a standard because now we have a mixture.